site/docs/components/avatar-group.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Avatar Group
+navTitle: Avatar Group
+summaryParagraph: Avatar Groups contains a list of companies, groups or users visually through profile images or initials
+tags: ["Profile picture", "Display picture"]
+githubLabels: ["component:avatar-group"]
+needToKnow:
+- An Avatar Group must have at least one Avatar provided
+- The `size` set in Avatar Group will supersede any passed in at the Avatar component level.
+- Use an avatar to help a user in the platform efficiently identify another person or an organization.
+- Use a profile avatar to represent an individual and an account avatar for an organization.
+- Company Avatars will not display initials and should be accompanied by an avatarSrc.
+- Has RTL support could have internationalisation support for the aria response in the render counter.
+demoStoryId: components-avatar-avatar-group--default-story
+headerImage: avatar
+health:
+  designed: true
+  documented: true
+  implemented: true
+  latestDesign: true
+  allVariants: true
+  responsive: true
+  internationalized: false
+  accessible: true
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FeZKEE5kXbEMY3lx84oz8iN%2F%25E2%259D%25A4%25EF%25B8%258F-UI-Kit-Heart%3Fnode-id%3D1929%253A14305" allowfullscreen></iframe>
+
+## To keep in mind
```

Comment:
The below feel `Avatar` specific, not sure how helpful or necessary they are for `AvatarGroup`.

Reply:
Good point, I've update the keen in mind to more AvatarGroup centric points